packages/core/src/order/order.ts

```diff
@@ -34,6 +34,15 @@ export default interface Order {
     taxes: Tax[];
     taxTotal: number;
     channelId: number;
+    fees?: OrderFee[];
```

Comment:
Is this supposed to an optional?

Reply:
I marked it as optional because fees are only included by default when coming from `api/storefront/orders/` if `PROJECT-5577.support_order_level_fee` is on.

I suppose that the field itself should always be in the payload, so it should be fine to have as mandatory.

Reply:
I think that makes sense to mark it mandatory.

Reply:
Marked as mandatory now.